src/components/nudge/Nudge.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../styles/variables';
+
+/**************************************
+ * Nudge
+ **************************************/
+
+.Nudge {
+    position: fixed;
```

Comment:
Another thing to consider is if the positioning, widths and heights all should be responsibility of the consuming app instead of this component. Can a nudge be within a section of the page as opposed to fixed to the whole window?

Reply:
i think based on the design language we're building here, we want it to be forced to the bottom right and fixed to the page.